diff --git a/src/config/main.cfg b/src/config/main.cfg
index 03573fc4..a69d1674 100644
--- a/src/config/main.cfg
+++ b/src/config/main.cfg
@@ -30,8 +30,8 @@ temp_dir_path = /tmp
 docker-mount-base-dir = /tmp/fact-docker-mount-base-dir
 
 [Logging]
-logFile=/tmp/fact_main.log
-mongoDbLogFile=/tmp/fact_mongo.log
+logFile=/var/log/fact/main.log
+mongoDbLogFile=/var/log/fact/mongo.log
 logLevel=WARNING
 
 
